paddle/fluid/platform/stream_callback_manager.h

```diff
@@ -14,11 +14,11 @@
 
 #pragma once
 
+#include <ThreadPool.h>
```

Comment:
the pre-commit would adjust the order automatically.

Reply:
fine.